src/main/scala/mesosphere/marathon/core/launchqueue/impl/ActorTaskTrackerUpdateSubscriber.scala

```diff
@@ -0,0 +1,25 @@
+package mesosphere.marathon.core.launchqueue.impl
```

Comment:
should be part of the tracker module. Everyone using an actor as a `TaskTrackerSubscriber` should use this code.